sys-devel/clang-crossdev-wrappers/clang-crossdev-wrappers-20.ebuild

```
@@ -0,0 +1,40 @@
+# Copyright 2022-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Symlinks to a Clang crosscompiler"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:LLVM"
+S=${WORKDIR}
+
+LICENSE="public-domain"
+SLOT="${PV}"
+PROPERTIES="live"
+
+RDEPEND="
+	llvm-core/clang:${SLOT}
+	llvm-core/lld:${SLOT}
+"
+
+src_install() {
+	local llvm_path="${EPREFIX}/usr/lib/llvm/${SLOT}"
```

```suggestion
	local llvm_path="/usr/lib/llvm/${SLOT}"
```

`into` (and `dosym`) add prefix automatically.

---

Done